README.md

```diff
@@ -44,7 +44,7 @@ Please note that the mock images only served as demonstration purpose.
 ## Get Involved in the Community
 
 Want help or want to help? 
-Join our [slack channel](https://bit.ly/2tFsN1X) and our [mailing list](https://groups.google.com/forum/#!forum/amundsen-dev)
+Join our [slack channel](https://github.com/lyft/amundsenfrontendlibrary#amundsen) and our [mailing list](https://groups.google.com/forum/#!forum/amundsen-dev)
```

Comment:
I think it's misleading that a link titled `slack channel` links back to our github page. I support the url only being in one spot though, so I think here we should give clear instruction like:
`Use the button in our [header](https://github.com/lyft/amundsenfrontendlibrary#amundsen) to join our slack channel...`

Reply:
updated